src/context/__tests__/TelegramContext.spec.js

```diff
@@ -101,6 +101,32 @@ it('get #client works', () => {
   expect(context.client).toBe(client);
 });
 
+describe('_getChatId', () => {
```

Comment:
Do we need to test private method?

Reply:
not pretty sure, but it does be counted into testing coverage.

Reply:
No, we shouldn't test private methods. We should test the public methods which use the private method you want to test.